near-sdk-sim/src/user.rs

```diff
@@ -35,10 +33,10 @@ type Runtime = Rc<RefCell<RuntimeStandalone>>;
 /// use near_sdk_sim::{to_yocto, account::AccessKey};
 /// use near_crypto::{InMemorySigner, KeyType, Signer};
 /// let master_account = near_sdk_sim::init_simulator(None);
-/// let account_id = "alice".to_string();
-/// let transaction = master_account.create_transaction(account_id.clone());
+/// let account_id = "alice";
+/// let transaction = master_account.create_transaction(account_id.parse().unwrap());
```

Comment:
Good attention to detail catching this in comment blocks 👍🏼

Reply:
I mean, rust doc tests catch this one, but I think I changed the ones outside of doc tests too, I'll double-check tomorrow